components/paragraph/bem.scss

```diff
@@ -4,7 +4,7 @@
  */
 
 .utrecht-paragraph {
-  color: var(--utrecht-document-color, inherit);
+  color: var(--utrecht-paragraph-color, inherit);
```

Comment:
Ik zou wel gebruik maken van `document` in tweede instantie:
`var(--utrecht-paragraph-color, var(--utrecht-document-color, inherit))`

Reply:
Dank je wel! Wordt verwerkt.